src/main/java/de/tum/in/www1/artemis/repository/StudentParticipationRepository.java

```diff
@@ -208,9 +222,9 @@ and not exists (select prs from participation.results prs
     List<StudentParticipation> findAllWithEagerSubmissionsAndEagerResultsAndEagerAssessorByExerciseId(long exerciseId);
 
     @Query("""
-            SELECT DISTINCT p FROM StudentParticipation p left join fetch p.submissions s left join fetch s.results r left join fetch r.assessor a
+            SELECT DISTINCT p FROM StudentParticipation p LEFT JOIN FETCH p.submissions s LEFT JOIN FETCH s.results r LEFT JOIN FETCH r.assessor a
             WHERE p.exercise.id = :#{#exerciseId}
-            AND NOT EXISTS (select prs from p.results prs where prs.assessor.id = p.student.id)
+            AND (p.testRun = FALSE OR p.testRun IS NULL)
```

Comment:
```suggestion
            AND p.testRun = FALSE
```

Reply:
fixed in 139c8fb547deadc2b10c0e497d91389dcb5f736c